config/config.yaml

```diff
@@ -15,11 +15,14 @@ detectors:
         service:
             hostname: https://localhost/api/v1/text/contents # full url / endpoint currently expected
             port: 8080
-            tls: caikit
+            tls: detector
         chunker_id: en_regex
         default_threshold: 0.5
 tls:
     caikit:
         cert_path: /path/to/tls.crt
         key_path: /path/to/tls.key
         client_ca_cert_path: /path/to/ca.crt
+    detector:
+        cert_path: /path/to/client.pem
+        self_signed: true
```

Comment:
I think the use here would be strictly testing purposes. Like this would be a way to test the mtls codepath from our side, like are we accepting and reading the ca certs and passing it to underlying library or not.